docs/spec/0.95.0/1_Overview.md

```diff
@@ -1,4 +1,7 @@
-# ElectionGuard Specification ![Version 0.95.0](https://img.shields.io/badge/Version-v0.95.0-green)
+# ElectionGuard Specification ![Version 0.95.0][green-badge-0.95.0]
 
 !!! info "Download Specification"
     No web version of this specification is available at this time. [Download specification here](https://github.com/microsoft/electionguard/releases/tag/v0.95.0)
+
+<!-- Links -->
+[green-badge-0.95.0]: https://img.shields.io/badge/Version-v0.95.0-green
```

Comment:
I think the badges are overkill, but the specification one is missed here.

Reply:
Yeah badges are probably overkill, but I felt it was good for consistency. Do you think I should remove it? Will update the spec link.